test/kubemark/cloud-provider-config.sh

```diff
@@ -15,3 +15,6 @@
 # limitations under the License.
 
 CLOUD_PROVIDER="${CLOUD_PROVIDER:-gce}"
+CONTAINER_REGISTRY="${CONTAINER_REGISTRY:-gcr.io}"
+PROJECT="${PROJECT:-}"
```

Comment:
We can set the project if we want to, but it's fine even otherwise as we fallback to `detect-project` setting $PROJECT.